lib/engine/action/base.rb

```diff
@@ -11,7 +11,7 @@ class Base
       attr_accessor :id, :user
 
       def self.from_h(h, game)
-        entity = game.get(h['entity_type'], h['entity'])
+        entity = game.get(h['entity_type'], h['entity']) || Player.new(h['entity'], h['entity'])
```

Comment:
what if we make ID 0 or nil?

Reply:
Yeah, that should happen. I wrote this before the change to id's and it went unnoticed when I rebased